crates/sncast/tests/e2e/account/create.rs

```diff
@@ -30,11 +30,9 @@ pub async fn test_happy_case() {
     ];
 
     let snapbox = runner(&args).current_dir(temp_dir.path());
-    let bdg = snapbox.assert();
-    let out = bdg.get_output();
+    let output = snapbox.assert();
 
-    let stdout_str =
-        std::str::from_utf8(&out.stdout).expect("failed to convert command output to string");
+    let stdout_str = output.as_stdout();
```

Comment:
why not use `stdout_matches` or `assert_stdout_contains`

Reply:
Because we also use `!stdout_str.contains` there, hence we need to convert to string anyway. Thought it would be reasonable to not use multiple approaches.